apps/backend/src/storage/storage.controller.ts

```diff
@@ -1,27 +1,59 @@
-import { Controller, Delete, Post, Query } from '@nestjs/common';
+import { Body, Controller, Post, Query, Param } from '@nestjs/common';
 import { StorageService } from './storage.service';
-import { ApiOperation, ApiQuery, ApiResponse } from '@nestjs/swagger';
+import {
+  CompleteMultiPartUploadData,
+  GeneratePreSignedUrlsPartsData,
+  ObjectKeyDto,
+} from './dto/storage-dto';
 
 @Controller('storage')
 export class StorageController {
   constructor(private readonly storageService: StorageService) {}
 
-  @ApiOperation({
-    summary:
-      'Create a S3 bucket for the ceremony. If the bucket already exists, it will be reused.',
-  })
-  @ApiQuery({ name: 'id', type: 'number' })
-  @ApiResponse({ status: 200, description: 'Return the bucket name.' })
-  @Post('/bucket')
-  createBucket(@Query('id') ceremonyId: number) {
+  @Post('ceremony/:id/bucket')
```

Comment:
Why `ceremony/...` could we keep `/bucket` for bucket endpoints and for the other ones `/storage/...`